### ISSUE TITLE
lodash dependency error

### ISSUE THREAD

**Issue by aslubsky** [scttcper/toastr-ng2#16]
Hi, thanks for great job.
Please add to docs how to "lodash" should be included into project.
I've error like this:
```
node_modules/toastr-ng2/portal/portal.d.ts(1,1): error TS2688: Cannot find type definition file for 'lodash'.
```
"toastr-ng2": "^1.4.0",

**Comment by scttcper** [comment]
Whoops. Fixed in 1.4.1. You do not need lodash.

**Comment by aslubsky** [comment]
Works fine. Thanks a lot
